examples/8b_benchmark_read_parallel.cpp

```diff
@@ -744,9 +744,9 @@ main( int argc, char *argv[] )
     MPI_Comm_size( MPI_COMM_WORLD, &input.m_MPISize );
     MPI_Comm_rank( MPI_COMM_WORLD, &input.m_MPIRank );
 
-    if (argc < 2) {
+    if (argc < 3) {
```

Comment:
Following the LGTM waarning: should the line 759 below read `>= 4` now?

Reply:
Yep, actually, we do not need that if in line 759